docs/init.md

```diff
@@ -1,15 +1,18 @@
-For new projects, it is possible to run `cz init`.
+To start using commitizen, the recommended approach is to run
 
-This command will prompt the user for information about the project and will
-configure the selected file type (`pyproject.toml`, `.cz.toml`, etc.).
-
-This will help you quickly set up your project with `commitizen`.
+```sh
+cz init
+```
 
-Currently `init` is detecting
+This command will ask you for information about the project and will
+configure the selected file type (`pyproject.toml`, `.cz.toml`, etc.).
 
-1. Commit convention rules (`name`).
-2. Version detection based on the existing tags (`version`).
-3. Tag format, if your tag convention uses, for example `v*` in front of the version. (`tag_format`)
+The `init` will help you with
 
-We hope to detect the files where the version is also repeated eventually,
-like `package.json` or `__version__`.
+1. Choose a convention rules (`name`)
+1. Choosing a version provider (`commitizen` or for example `Cargo.toml`)
+1. Detecting your project's version
+1. Detecting the tag format used
+1. Choosing a version type (`semver` or `pep440`)
+1. Whether to create the changelog automatically or not during bump
+1. Whether you want to keep the major as zero while building alpha software.
```

Comment:
```suggestion
2. Choosing a version provider (`commitizen` or for example `Cargo.toml`)
3. Detecting your project's version
4. Detecting the tag format used
5. Choosing a version type (`semver` or `pep440`)
6. Whether to create the changelog automatically or not during bump
7. Whether you want to keep the major as zero while building alpha software.
```

nitpick: even though I know markdown will handle that for us, I still think it would be more readable haha